src/pages/index.js

```diff
@@ -65,35 +129,43 @@ const previewModalDeleteBtn = previewModal.querySelector(
 const cardTemplate = document.querySelector("#card-template");
 const cardList = document.querySelector(".cards__list");
 
-function getCardElement(data) {
-  const cardElement = cardTemplate.content
-    .querySelector(".card")
-    .cloneNode(true);
+let selectedCard, selectedCardId;
 
-  const cardNameEl = cardElement.querySelector(".card__title");
-  const cardImgEl = cardElement.querySelector(".card__image");
+function getCardElement(data) {
+  const cardElement = cardTemplate.content.cloneNode(true).firstElementChild;
+  const cardImage = cardElement.querySelector(".card__image");
+  const cardTitle = cardElement.querySelector(".card__title");
   const cardLikeBtn = cardElement.querySelector(".card__like-btn");
   const cardDeleteBtn = cardElement.querySelector(".card__delete-btn");
 
-  cardNameEl.textContent = data.name;
-  cardImgEl.src = data.link;
-  cardImgEl.alt = data.name;
+  const isLiked = localStorage.getItem(`card-${data._id}-isLiked`) === "true";
```

Comment:
```js

 const isLiked = localStorage.getItem(`card-${data._id}-isLiked`) === "true";
```

Please, don't use `localStorage` for liking because everything is inside `data.isLiked`


```js
  const isLiked = data.isLiked

  if (isLiked) {
    cardLikeBtn.classList.add("card__like-btn_liked");
  }


```